app/views/batch_contexts/_batch_context.html.erb

```diff
@@ -24,8 +24,8 @@
 <dl class="dl row">
   <dt class="col-sm-3">Content Structure</dt>
   <dd class="col-sm-9"><%= batch_context.content_structure %></dd>
-  <dt class="col-sm-3">Bundle Directory</dt>
-  <dd class="col-sm-9"><%= batch_context.bundle_dir %></dd>
+  <dt class="col-sm-3">Staging location</dt>
```

Comment:
no idea if @andrewjbtw cares if this is a capital L in Location ...

Reply:
i just capitalized it for consistency